README.md

```diff
@@ -195,6 +198,21 @@ try {
 }
 ```
 
+<a name="scav" />
+
+### Add custom request attributes
+
+You can use the SDK to add custom request attributes to current traced service. Custom request attributes allow you to do easier/better filtering of your requests in Dynatrace.
```

Comment:
"to current traced service" => "to the current traced service"
"to do easier/better filtering" => "to do advanced filtering"